qcodes/instrument_drivers/QuTech/IVVI.py

```diff
@@ -183,7 +183,45 @@ def _get_version(self):
             mes = self.ask(bytes([3, 4]))
             ver = mes[2]
             return ver
-
+    
+    def linsweep(self,start,end,points):
```

Comment:
is linsweep a good name? wouldn't the name of this function make the impression that it sweeps some voltage due to the word `sweep` in it? perhaps there is a better name that indeed refers to `numpy.linspace` like `dac_linspace` or `linspace_dac` or else?

Reply:
Taking on board your comments; I've put forward a new version.
I think I have addressed everything. But if I missed something it's just a mistake.